resources/server/boot/boot.service.ts

```diff
@@ -0,0 +1,86 @@
+import BootDb, { _BootDb } from './boot.db';
+import { bootLogger } from './boot.utils';
+import { config } from '../server';
+
+const { identifierColumn, phoneNumberColumn } = config.database;
+const requiredDbColumns = [identifierColumn, phoneNumberColumn];
+
+const frameworkDependencies = {
+  es_extended: ['esx-npwd'],
```

Comment:
Should this not be 'es_extended'

Reply:
The prettier config auto corrects it

Reply:
Well qb-core has '' so kind of weird es_extended wouldn’t

Reply:
@RockySouthpaw It's because the `_` is valid in a object property, while `-` is not :/